packages/cli/src/resource/commands.json

```diff
@@ -144,10 +144,14 @@
     "description": "Run the publish stage in teamsapp.yml."
   },
   "uninstall": {
-    "description": "Remove an acquired M365 App.",
+    "description": "Clean up resources associated with Manifest ID, Title ID or an environment in Teams Toolkit generated project. Resources including application registration in Teams Developer Portal, bot registration in Bot Framework Portal and uploaded custom applications in Microsoft 365 apps.",
```

Comment:
Proposed: Clean up resources associated with Manifest ID, Title ID, or an environment in Teams Toolkit generated project. Resources include app registration in Teams Developer Portal, bot registration in Bot Framework Portal, and uploaded custom apps in Microsoft 365 apps.